config/locales/ru/admin.yml

```diff
@@ -98,37 +98,37 @@ ru:
         pending: "Нерешенный Вопрос"
       views: # This is the number of viewers or visits. Used to indicate the number of video views
         one: "%{count} Просмотр" # Eg. 1, 21, 31, 41, 51, 61, 71, 81, 101, 1001, …
-        few: "%{count} Views" # Eg. 2~4, 22~24, 32~34, 42~44, 52~54, 62, 102, 1002, … # TODO: Translate this
+        few: "%{count} Просмотра" # Eg. 2~4, 22~24, 32~34, 42~44, 52~54, 62, 102, 1002, … 
         many: "%{count} Просмотров" # Eg. 0, 5~19, 100, 1000, 10000, 100000, 1000000, …
-        other: "%{count} Views" # Eg. 0.0~1.5, 10.0, 100.0, 1000.0, 10000.0, 100000.0, 1000000.0 # TODO: Translate this
+        other: "%{count} Просмотра" # Eg. 0.0~1.5, 10.0, 100.0, 1000.0, 10000.0, 100000.0, 1000000.0 
 
     messages:
       loading: "Загружается"
       errors: "Ошибки"
-      no_translations_found: "Все статьи типа %{pages} уже переведены"
+      no_translations_found: "Все статьи типа \"%{pages}\" уже переведены"
       make_public: "Сделать %{page} видимой для пользователей"
       maintenance_mode: "Сайт в разработке, возвращайтесь через некоторое время."
       confirm_destroy: "Вы уверены, что хотите удалить это? Действие невозможно будет отменить!"
-      confirm_discard_draft: "Are you sure you want to discard all changes shown on this page? This cannot be undone!" # TODO: Translate
+      confirm_discard_draft: "Вы уверены, что хотите отменить правки, показанные на этой странице? Действие невозможно будет отменить!" 
       confirm_unsaved_changes: "Подождите! Несохраненные данные будут потеряны. Вы уверены, что хотите покинуть страницу?"
       drop_files_here: "Переместите файлы сюда"
       unpublished_changes_notice: "%{page} имеет несохраненные изменения. Изменения не видны пользователям, пока они не утверждены администратором."
       pending_invitation_notice: "%{person} получил приглашение стать редактором сайта, но ещё не ответил."
       draft_notice: "Вы просматриваете черновик %{page}. Эта версия не видна пользователям, пока они не утверждена администратором."
       draft_contributed_by: "Этот черновик содержит изменения, сделанные %{people}."
       text_too_short: "Должно быть по крайней мере %{count} букв"
```

Comment:
This phrase should start with a lower case, because it will be appended to the name of a field.
eg. Title "must have at least %{count} letters"

Reply:
Changed the wording to eg. Title ": must have at least %{count} letters"
That's because "must have" is spelled differently depending on the gender of the subject, i.e. title is masculine, but article is feminine. 
The way I wrote it accommodates for all genders (we also have neutral in Russian)